indico/modules/rb/client/js/common/rooms/edit/RoomEditOptions.jsx

```diff
@@ -0,0 +1,132 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useMemo} from 'react';
+import {Dropdown, Form, Header, Tab} from 'semantic-ui-react';
+import _ from 'lodash';
+import {FieldArray} from 'react-final-form-arrays';
+import PropTypes from 'prop-types';
+import {Translate} from 'indico/react/i18n';
+import {FinalCheckbox, FinalField, FinalInput} from 'indico/react/forms';
+import EquipmentList from './EquipmentList';
+import DailyAvailability from './DailyAvailability';
+import NonBookablePeriods from './NonBookablePeriods';
+
+export default function RoomEditOptions({active, showEquipment, globalAttributes}) {
+  const attributeTitles = useMemo(
+    () => (globalAttributes ? _.fromPairs(globalAttributes.map(x => [x.name, x.title])) : []),
+    [globalAttributes]
+  );
+
+  return (
+    <Tab.Pane active={active}>
+      <Header>
+        <Translate>Availability</Translate>
+      </Header>
+      <Form.Group>
+        <FinalField name="bookable_hours" component={DailyAvailability} isEqual={_.isEqual} />
+        <FinalField name="nonbookable_periods" component={NonBookablePeriods} isEqual={_.isEqual} />
+      </Form.Group>
+      <Header>
+        <Translate>Equipment and custom attributes</Translate>
+      </Header>
+      <Form.Group>
+        {showEquipment && (
+          <FinalField
+            name="available_equipment"
+            component={EquipmentList}
+            isEqual={_.isEqual}
+            componentLabel={Translate.string('Add new equipment')}
+          />
+        )}
+        {globalAttributes && (
```

Comment:
Don't you want to check for `globalAttributes.length > 0`? `[]` evaluates to `true`

Reply:
The purpose of `globalAttributes &&` was to hide them if they did not load. But I'm guessing we probably don't need to show them if we did not define any, globally. So, maybe `globalAttributes && globalAttributes.length > 0`.